### PR TITLE
[gapi.youtube] updated GoogleApiYouTubeVideoResource "thumbnails" object

### PR DESCRIPTION
**Pull request by orizens**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://developers.google.com/youtube/v3/docs/thumbnails#properties>>
- [ ] Increase the version number in the header if appropriate.

### CODE REVIEW

File: types/gapi.youtube/index.d.ts [hunk 1]
```diff
@@ -1894,7 +1902,7 @@ interface GoogleApiYouTubeVideoResource {
         /**
          * A map of thumbnail images associated with the video. For each object in the map, the key is the name of the thumbnail image, and the value is an object that contains other information about the thumbnail.
          */
-        thumbnails: GoogleApiYouTubeThumbnailItemResource[];
+        thumbnails: GoogleApiYouTubeThumbnailResource;
```
Comment (andy-ms) [inline types/gapi.youtube/index.d.ts]: The type here seems to be contradicted by the documentation, which would imply a `Map<string, X>` or `{ [key: string]: X }`.

Comment (orizens) [inline types/gapi.youtube/index.d.ts]: thanks - i updated the definition.

Comment (andy-ms) [inline types/gapi.youtube/index.d.ts]: I don't see the change here, did you remember to push?

Comment (orizens) [inline types/gapi.youtube/index.d.ts]: i used github's edit in place - i can see that it didn't update the pr.

**Comment by yuit** [comment]
Thanks for your contribution. This PR has unaddressed comment and cannot be merged in at the time being. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Pull request closed without merging**